php/Terminus/Commands/WorkflowsCommand.php

```diff
@@ -150,6 +152,53 @@ function($operation) {
     }
   }
 
+  /**
+   * Streams new and finished workflows to the console
+   *
+   * ## OPTIONS
+   * [--site=<site>]
+   * : Site from which to list workflows
+   *
+   * @subcommand watch
+   */
+  public function watch($args, $assoc_args) {
+    $site = $this->sites->get(Input::sitename($assoc_args));
+
+    $this->logger->info('Watching workflows...');
+    while (true) {
+      $last_checked = time();
+      sleep(WORKFLOWS_WATCH_INTERVAL);
+
+      $site->workflows->fetchWithOperations();
+      $workflows = $site->workflows->all();
+      foreach ($workflows as $workflow) {
+        if ($workflow->get('created_at') > $last_checked) {
+          $this->logger->info(
+            '{uuid} Started {description} ({environment})',
+            array(
+              'uuid' => $workflow->id,
+              'description' => $workflow->get('description'),
+              'environment' => $workflow->get('environment'),
+              'created_at'  => $workflow->get('created_at')
```

Comment:
Perhaps
`'created_at'  => date('Y-m-dTH:i:s', strtotime($workflow->get('created_at')));`
would be useful?